packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelSummary.tsx

```diff
@@ -126,25 +134,34 @@ function TotalGasFees() {
    *  only show child chain native currency
    *  x XAI
    */
-  return (
-    <>
-      {isDepositMode && (
-        <span className="tabular-nums">
+  if (isDepositMode) {
```

Comment:
Moved things around here to accommodate for the new UI but the logic should be the same. Please double check.